lib/create-script.js

```diff
@@ -33,6 +33,7 @@ export default async function createScript({ description, scriptOptions }) {
 
   const content = [
     `
+      // @ts-check
```

Comment:
maybe add a new line?

```suggestion
      // @ts-check
      
```